static/js/disease/chart.ts

```diff
@@ -17,14 +17,264 @@ import * as d3 from "d3";
 import _ from "lodash";
 const SVGNS = "http://www.w3.org/2000/svg";
 const XLINKNS = "http://www.w3.org/1999/xlink";
+const MARGIN = { top: 30, right: 30, bottom: 90, left: 160 };
+// graph specific dimensions
+const GRAPH_HEIGHT = 400;
+const GRAPH_WIDTH = 760;
+// number to select top data points for large data
+const NUM_DATA_POINTS = 10;
+// number by which y axis domain is increased/decreased for y scale to fit all error bars well
+const Y_AXIS_LIMIT = 0.5;
+// dimension of tissue legend dots
+const LEGEND_CIRCLE_RADIUS = 4;
+// tooltip constant for all charts
+const TOOL_TIP = d3.select("#main").append("div").attr("class", "tooltip");
+// horizontal barcharts - default brightness
```

Comment:
i imagine this is being used for all charts (not just horizontal).

Reply:
Yes, it's for all charts. Removed the word `horizontal`!